source/test/contentscript/api.test.ts

```diff
@@ -431,6 +431,16 @@ function additionalTests() {
 
     await closeTab(await tabIdPromise);
   });
+
+  test("`frameId: allFrames` messages every frame in tab, receives only one response", async (t) => {
+    // `chrome.*.sendMessage` can exclusively receive one response
```

Comment:
Note.

Just like `runtime.sendMessage`, even if multiple contexts receive the message, only one can respond. However, unlike `runtime.sendMessage`, all the frames will receive the message even if one frame already handled/responded to it.

If `getMethod` is used instead of `getNotifier`, only one frame can respond.